vector/src/main/java/im/vector/app/features/home/room/list/SpaceRoomListSectionBuilder.kt

```diff
@@ -129,7 +129,7 @@ class SpaceRoomListSectionBuilder(
                 sections, activeSpaceAwareQueries,
                 R.string.invitations_header,
                 true,
-                RoomListViewModel.SpaceFilterStrategy.NONE
+                RoomListViewModel.SpaceFilterStrategy.NOT_IF_ALL
```

Comment:
I still do not get the meaning of `NOT_IF_ALL` ha ha

Reply:
Documented in SpaceFilterStrategy

````
 /**
         * Special case when we don't want to discriminate rooms when current space is null.
         * In this case return all.
         */
        NOT_IF_ALL,
````

Reply:
Yeah I've seen the comment, thanks, but it's still hard to understand to me ha ha